Cargo.toml

```diff
@@ -13,5 +13,6 @@ members = [
     "manufacturing-server",
     "manufacturing-client",
 
+    "serviceinfo-api-dev-server",
```

Comment:
I think I forgot why we call it `-dev-`?

Reply:
Because this implementation is incredibly rudimentary (it doesn't allow different serviceinfo for different device), and it's purely here for the integration tests.
For live deployments, people are expected to implement the API themselves.

Reply:
right, thanks Patrick!